race.test.js

```diff
@@ -13,10 +13,15 @@ async function openParallelCalls(n, id) {
     const results = await Promise.all(responses.map(res => res.text()));
 
     let firstResult;
+    // remains true if all zoom call links match
     let allMatch = true;
     results.forEach((result, idx) => {
+
```

Comment:
The method you use here to detect the duplicates is not very straight-forward, relying on higher-scoped mutable variables...  Can you think of a more elegant way of checking?  One example might be to [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) the raw responses to the zoom urls like you do below, but them [collect them to a set](https://stackoverflow.com/a/49438569), and then check the # of entries.